index.bs

```diff
@@ -224,7 +224,7 @@ In order to report a {{Gamepad/mapping}} of {{GamepadMappingType/"xr-standard"}}
     </tr>
     <tr>
       <td>buttons[1]</td>
-      <td>Secondary Button/Grip Trigger</td>
+      <td>Secondary Button/Squeeze Trigger</td>
```

Comment:
Per https://github.com/immersive-web/webxr-gamepads-module/issues/8#issuecomment-533036085, I'd just call this row "Squeeze Press" (or maybe just "Squeeze") and the previous row "Primary Press".

Mentioning button/trigger may still be too presumptuous if the controller has a force sensor for a given input, like the Valve Index.  Just talking about 4 kinds of presses keeps things pretty straightforward, perhaps with an example:
* Primary Press (e.g. button/trigger)
* Squeeze Press
* Touchpad Press
* Thumbstick Press

Reply:
Ah I hadn't previously noticed that these say "press".  That doesn't actually seem quite right because its more than just the pressed state.  There's value (in the range 0-1), touch, and press.

Also I think we were intending a restriction that the first entry must be a trigger not a face button.

So perhaps just:
* Primary Trigger
* Primary Squeeze
* Primary Touchpad
* Primary Thumbstick

WDYT?  Too confusing?

Not that I think it's worth reevaluating at this time, but boy it seems like maybe we should have considered just exposing the touchpad/thumbstick as having three axes instead of two.

Reply:
I prefer avoiding "press", "Primary Trigger", etc makes sense

Reply:
The text above describes an `"xr-standard"` controller this way, allowing it to have either a primary button or a trigger:

> It MUST have at least one primary button, often a trigger, separate from any touchpads or thumbsticks.

For now, maybe we should just have this say "Primary Trigger/Button" and open a separate issue to discuss whether to now restrict `"xr-standard" controllers further to just primary triggers.

Reply:
Seems reasonable.  Should we continue this portion of the conversation in #8?  Or a new issue?

Reply:
Let's make a new issue - #8 is fully fixed by this change.

Reply:
Filed #17 to track the button 0 trigger vs. primary trigger/button question.